src/containers/Home/index.js

```diff
@@ -1,17 +1,84 @@
-import React from "react";
-import { Wrapper } from "../../components/Wrapper/";
-import Header from "../Header";
-import Enssemble from "../../components/Enssemble";
+import React from 'react';
+import { Wrapper } from '../../components/Wrapper/';
+import { connect } from 'react-redux';
+
+import HorlogeSvg from "../../components/SVG/HorlogeSvg";
+import ChairSvg from "../../components/SVG/ChairSvg";
+import flatBackground from "../../assets/appartement4.png";
+import { Link } from "react-router-dom";
 
 class Home extends React.Component {
+  componentWillMount = () => {
+    this.setState({
+      screenHeight: window.innerHeight + 'px',
+      screenWidth: window.innerWidth + 'px',
+    });
+  };
+
   render() {
     return (
-      <Wrapper>
-        <Header />
-        <Enssemble />
+      <Wrapper
+        height={this.state.screenHeight}
+        width={this.state.screenWidth}
+        position="relative"
+        margin="auto"
+      >
+        <Wrapper
+          position="absolute"
+          height={this.state.screenHeight}
+          width={this.state.screenWidth}
+        >
+          <img src={flatBackground} alt="flat" height="100%" width="100%" />
+        </Wrapper>
+
+        <Wrapper
+          position="absolute"
+          top="40%"
+          left="20%"
+          action={this.shouldRenderModal}
+        >
+          <Link to={{ pathname: "/modal", initialColor: this.props.chair }}>
```

Comment:
OK pour passer des props (en l'occurence `initialColor`) à une route avec cette méthode mais, puisque la prop n'est pas dans l'URL, on n'aura pas `initialColor` si on charge cette URL (en cas de bookmark ou de partage de l'URL).

La solution à ça serait de passer la couleur dans l'URL : ``to=`/modal?initialColor=${this.props.chair}` ``.